.gitignore

```diff
@@ -3,9 +3,6 @@
 *.o
 *.out
 *.log
-\#*
-\*.*
```

Comment:
Atom behaves strangely when these exist in any gitignores, so we can't use them until fixed